tests/test_acl_backend.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
+
+import six
+import pytest
+from test_utils.factories import (
+    UserFactory,
+    CourseAccessGroupFactory,
+    CourseOverviewFactory,
+    MembershipFactory,
+    OrganizationFactory,
+    GroupCourseFactory,
+)
+from organizations.models import OrganizationCourse, UserOrganizationMapping
+from course_access_groups.acl_backends import default_backend
+from test_utils import patch_site_configs
+
+@pytest.mark.django_db
+class TestAclBackend(object):
+    """
+    Tests for the access control backend.
+
+    Testing the integration point with Open edX.
+
+    Platforms' `default_has_access` is always checked via this backend.
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_models(self):
```

Comment:
Why `def init_models` instead of `def setup`

Reply:
Done.